### PR TITLE
62/view component

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Part of #62

Follow up to #90 comments

- Split component into index and view
- Added storybook
screenshot_2021-01-28_15-19-08

See https://pr91--gpui.review.gnosisdev.com/storybook/?path=/story/explorer-orderwidget--order-found

Text color and font still look odd though. Am I missing some decorator to apply the theme properly?

### CODE REVIEW

File: src/apps/explorer/components/OrderWidget/OrderWidget.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    '0x04dca25f59e9ac744c4093530a38f1719c4e0b1ce8e4b68c8018b6b05fd4a6944e1dcf2a009df2d5932f7c034b4a24da0999f9309dd5108d51d54236b605ed991c',
+}
+
+export const OrderFound = Template.bind({})
+OrderFound.args = { ...defaultProps, order }
+
+export const OrderLoading = Template.bind({})
+OrderLoading.args = { ...defaultProps, isLoading: true }
+
+export const OrderNotFound = Template.bind({})
```
Comment (W3stside) [inline src/apps/explorer/components/OrderWidget/OrderWidget.stories.tsx:48]: same as line 42 no?

Comment (anxolin) [reply src/apps/explorer/components/OrderWidget/OrderWidget.stories.tsx:48]: I think in this one order is `undefined`, htat the difference

File: src/apps/explorer/components/OrderWidget/index.tsx [hunk 2]
```diff
@@ -1,44 +1,31 @@
...
-  return (
-    <Wrapper>
-      <h2>Order details</h2>
-      {order ? <OrderDetails order={order} /> : <p>Order not found</p>}
-      <h2>Order fills</h2>
-      <p>No fills</p>
-      {error && <p>{error}</p>}
-    </Wrapper>
-  )
+  return <OrderWidgetView order={order} isLoading={isLoading} error={error} />
```
Comment (anxolin) [inline src/apps/explorer/components/OrderWidget/index.tsx:30]: I vote against this naming convention. 
I prefer to call the dumb components without any suffix, they are just normal components.

First naming it `OrderWidgetView` u imply there's a 1<>1 relationship between OrderWidget and OrderWidgetView, which might be true here at least for now, although is not sth we always want to do. 

Another concern would be that the names get unnecesarilly complex. `MyCoolComponentWidgetViewStyledWrapperHoH` (disclaimer: some names in this film may or may not have been dramatized)

**Review by anxolin — APPROVED** [review]

**Comment by anxolin** [comment]
Sorry I didn't mention this before, but the path should be "/orders/<id>" not "/order/<id>"

**Review by W3stside — APPROVED** [review]

**Comment by anxolin** [comment]
@alfetopito @W3stside  notice I approve, to not stop the PR for this, but I'm interested discussing https://github.com/gnosis/gp-ui/pull/91#discussion_r566684624

File: src/apps/explorer/components/OrderWidget/index.tsx (hunk 2 above)
Comment (alfetopito) [reply src/apps/explorer/components/OrderWidget/index.tsx:30]: What do you suggest?

Comment (W3stside) [reply src/apps/explorer/components/OrderWidget/index.tsx:30]: let's discuss [here](https://github.com/gnosis/gp-ui/issues/92)

Comment (anxolin) [reply src/apps/explorer/components/OrderWidget/index.tsx:30]: Good call opening the issue!

**Pull request merged**